name,total_ops,threads,spent,throughput,latency
FxDashMap,512,1,5523003998,92.70317388606026,10787117
FxDashMap,512,2,3397786470,150.686337861602,13272603
FxDashMap,512,3,54132540,9458.266691346831,317182
FxDashMap,512,4,1825279795,280.50494033984523,14259998
FxDashMap,512,5,83374734,6140.949127345941,814206
FxDashMap,512,6,208055,2460887.7460286943,2438
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 61, kind: ConnectionRefused, message: "Connection refused" }', src/adapters/dashmap.rs:20:61
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
